tests/integration/test_recipe.py

```diff
@@ -826,6 +867,8 @@ def test_custom_preproc_order(tmp_path, patched_datafinder, config_user):
           custom:
```

Comment:
Shouldn't we test here for a custom order that has both extract_timerange (by means of the dataset keys) and extract_time (by means of the standard preprocessor)?

Reply:
Yes, I added a test for that!